README.md

```diff
@@ -1,14 +1,40 @@
-# Python GUI for M1M3 Support System
+# Python EUI/GUI for M1M3 Support System
+
+Engineering / User Graphical Interface for cRIO systems. Provides generics GUI
+classes as well as SALobj specific classes.
+
+## Directory structure
+
+Under python/lsst/ts/cRIOpy, the directories are:
+
+* __AirCompressor__ widget for M1M3-ComCam air compressor
+* __GUI__ Generic GUI widgets
+  * __ActuatorsDisplay__ contains widgets to plot actuators
+  * __CustomLabels__ contains plenty of QLabel childs to be used building up user interface
+  * __SAL__ contains SAL bindend Labels. Please see __SALComm__ for details
+  * __TimeChart__ contains QtCharts.QChart subclasses to build up real-time graphs
+* __M1M3__ widgets for M1M3 support system
+* __M1M3TS__ widgets for M1M3 thermal system
+* __VMS__ graphs for [ts_VMS](https://github.com/lsst-ts/ts_VMS)
 
 ## Applications
 
 Applications are located in bin directory. Before running those, please make
 sure that required Python packages (in python directory) are available:
 
+```
```

Comment:
You could add the syntax hint of language (see the section of "Syntax Highlighting"):
https://www.markdownguide.org/extended-syntax/